index.js

```diff
@@ -0,0 +1,84 @@
+//alert("Test")
+
+//KEY
+//1 - rock
+//2 - paper
+//3 - scissors
+
+var c = 0;
```

Comment:
Don't use vague variable names like this - this should be something like computerScore.